modules/@apostrophecms/schema/ui/apos/lib/conditionalFields.js

```diff
@@ -0,0 +1,181 @@
+// Evaluate the external conditions found in each field
+// via API calls - made in parallel for performance-
+// and store their result for reusability.
+// `schema` - the fields schema
+// `docIdOrContextDocId` - the current docId
```

Comment:
If we're not going to explain what it means when it's not just the doc id, we may as well just call it `docId`. Or `contextId` if it's not always a doc.